ramls/holdingsrecordssources.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A collection of holdings records sources",
+  "type": "object",
+  "properties": {
+    "holdingsRecordsSources": {
```

Comment:
Personally, I would be tempted to name this `sources` (or maybe `holdingsSources`). What do you think?